ngx-fudis/projects/ngx-fudis/src/lib/ngx-fudis.module.ts

```diff
@@ -89,6 +89,7 @@ import { LanguageBadgeGroupComponent } from './components/language-badge-group/l
 import { LanguageBadgeComponent } from './components/language-badge-group/language-badge/language-badge.component';
 import { FudisTranslationService } from './utilities/translation/translation.service';
 import { FudisLanguageBadgeGroupService } from './components/language-badge-group/language-badge-group.service';
+import { BreadcrumbsComponent } from './components/breadcrumbs/breadcrumbs.component';
```

Comment:
In addition to importing the component here, we need to export it also in `public-api.ts`. That is how the component is exposed to the public npm package (vs. internal component).